sys/sys/proc.h

```diff
@@ -1204,6 +1204,13 @@ curthread_pflags2_restore(int save)
 	curthread->td_pflags2 &= save;
 }
 
+static __inline bool
+kstack_contains(struct thread *td, vm_offset_t va, size_t len)
+{
+	return (va >= td->td_kstack && va + len >= va &&
```

Comment:
I feel like surely there must already be a helper function for "range is subset of another range" somewhere, but a quick search doesn't show me anything.

Reply:
No, it's open-coded in various places though. :-/